### PR TITLE
1038813: Need to update Getting Started Documentation for BlockEditor

### PR DESCRIPTION
**Pull request by VikramVaratharajan-SF**
### Bug description
The Getting Started User Guide was not sufficiently clear and detailed for customers to easily understand the setup, configuration, and usage flow of the components.

### Root cause
The documentation contained limited step-by-step explanations and lacked refinement in structure, readability, and customer-focused guidance.

### Solution description
Refined and updated the Getting Started User Guide by improving content clarity, organizing instructions more effectively, and enhancing the overall readability and usability for customers.

### Code Studio usage(Mandatory)
* Code Studio used in this PR/MR?
    - [ ] Yes
    - [x] No
* If `Yes`: Primary use (choose one)
    - [ ] Generate new code
    - [ ] Refactor/improve existing code
    - [ ] Tests
    - [ ] Bug fix / debugging help
    - [ ] Docs / comments
    - [ ] Review assistance (explanations/summaries)
    - [ ] Other: 

* Outcome
    - [ ] Saved time
    - [ ] Neutral
    - [ ] Cost time
* If “Cost time” explain in short (1 or 2 lines):

MD file:

### Impact assessment
* [x] Low - Affects a single feature with minimal user impact
* [ ] Medium - Affects multiple features or has moderate user impact
* [ ] High - Critical functionality or significant user impact

### Reason for not identifying earlier
Failed to check with two chatUI instances.
     
### Areas tested against this fix
* [ ] Tested using standard test cases
* [ ] Tested against feature matrix. [Centralized location](https://syncfusion.sharepoint.com/:f:/r/sites/Blazor/Shared%20Documents/Feature%20Matrix%20-%20Documents?csf=1&web=1&e=UToeuF)
* [x] NA

### Breaking changes
* [ ] Yes (Tag `breaking-issue`)
* [x] No
 
If yes, provide breaking commit details link and migration guidance.
 
### Regression testing
* [x] Verified fix doesn't reintroduce previous bugs
* [x] Checked edge cases and error scenarios

### Action taken to prevent recurrence
* [x] Added/updated unit tests
* [ ] Other (specify): _________________
* [ ] NA

### Automation status
* [ ] BUnit (provide PR link: _________________)
* [ ] Playwight (provide PR link: _________________)
* [x] NA

### Cross-platform verification
* [ ] Blazor Server
* [ ] Blazor WASM
* [x] NA
 
### Related issues
Is this issue present in EJ2 or other components?
* [ ] Resolved in EJ2 (PR link: _________________)
* [x] Created task for EJ2 (Task link: https://dev.azure.com/EssentialStudio/Ej2-Web/_workitems/edit/1019670)
* [ ] Needs attention in other components (tag `needs-attention-coreteam`)
* [ ] NA

### Output screenshots
NA

### API changes
* [ ] New API added (API Review task link: _________________)
* [ ] Existing API renamed/modified (API Review task link: _________________)
* [x] No API changes

### Performance verification
* [ ] Verified no memory leaks introduced
* [ ] Verified no performance degradation
* [x] Not applicable

### Reviewer Checklist
* [ ] Reviewed the provided Code Studio usages related information.
* [x] Code changes follow component guidelines
* [x] All provided information reviewed and verified
* [ ] Solution addresses the root cause effectively

### CODE REVIEW

File: ej2-asp-core-mvc/block-editor/EJ2_ASP.MVC/getting-started.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 layout: post
 title: Getting Started with ASP.NET MVC BlockEditor Control | Syncfusion
 description: Checkout and learn about getting started with Syncfusion Essential ASP.NET MVC BlockEditor control, its elements, and more.
+canonical_url: "https://www.syncfusion.com/aspnet-mvc-ui-controls/block-editor"
 platform: ej2-asp-core-mvc
```
Comment (keerthikarankv) [inline ej2-asp-core-mvc/block-editor/EJ2_ASP.MVC/getting-started.md:6]: add current SDK path link and ensure in all places

File: ej2-asp-core-mvc/block-editor/EJ2_ASP.NETCORE/getting-started.md
Comment (keerthikarankv) [inline ej2-asp-core-mvc/block-editor/EJ2_ASP.NETCORE/getting-started.md:47]: could use Individual BlockEditor nuget and scripts
